### ISSUE TITLE
Make composer.json valid

### ISSUE THREAD

**Issue by behzad1q** [sunra/php-simple-html-dom-parser#32]
By running `composer validate`
we get following.

```
"./composer.json" does not match the expected JSON schema:
   - authors[0].name : The property name is required
```
Note: I have the patch file but do not have access push to repository to create pull request

**Comment by sunra** [comment]
I added the Name field.

"Note: I have the patch file but do not have access push to repository to create pull request"
What is the problem, how do you think?

**Comment by behzad1q** [comment]
Looks good to me and the validation warning gone. 
Just I think there are more spaces before "name": "S.C. Chen", line left in the composer.json file.
About the my permission to update the file I will work it out.

**Comment by sunra** [comment]
"line left in the composer.json file"
@behzad1q I know about it

**Comment by behzad1q** [comment]
Still our build is failed because the changes is not merged to 1.5.1 branch and 1.5.1 is the latest and loads by composer require.
